jobs/ar-reminder/pyproject.toml

```diff
@@ -34,9 +34,82 @@ requests-mock = "^1.12.1"
 
 [tool.poetry.group.dev.dependencies]
 pre-commit = "^3.7.0"
+black = "^23.12.1"
 faker = "^24.4.0"
 lovely-pytest-docker = "^0.3.1"
+pylint = "^3.0.3"
+bandit = "^1.7.6"
+flake8-pyproject = "^1.2.3"
+isort = "^5.13.2"
 
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.isort]
```

Comment:
try replacing isort config with (and rerun isort locally):

[tool.isort]
atomic = true
profile = "black"
line_length = 120
skip_gitignore = true
skip_glob = ["devops"]
known_first_party = ["ar_reminder", "business_ar_api"]

Reply:
Just pushed changes but the error still exist, added skip_isort instructed by Patrick